Bottom: a52fff43a00ef7782f2bed16ecc75238cfab664d
Top:    7cc083b4c8358004bd13184fa90ac2cdb4c48c41
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-02 13:02:35 +0000

Refresh of hid-common-fix-including-draw_

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index f6fbe71..02473be 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -4,6 +4,7 @@
 #include "data.h" /* For global "PCB" variable */
 #include "rotate.h" /* For RotateLineLowLevel() */
 #include "polygon.h"
+#include "draw_helpers.h"
 
 
 static void
